pages/import.html

```diff
@@ -13,6 +13,19 @@
       </label>
       <button class="ui button" id="uploadButton" type="submit">Submit</button>
     </form>
+
+<!--      <br>
+      <label>Vorname:</label>
+      <label id="first-name"></label>
+      <br>
+      <label>Nachname:</label>
+      <label id="last-name"></label>
+      <br>
+      <label>Geschlecht:</label>
+      <label id="gender"></label>-->
+
+      <div id="qr-content"></div>
+
```

Comment:
Das kannst du entfernen, wird sonst in der website html auch dargestellt (ist unsichtbar)